mongodbatlas/resource_mongodbatlas_privatelink_endpoint_serverless.go

```diff
@@ -8,11 +8,12 @@ import (
 	"strings"
 	"time"
 
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+
```

Comment:
you need to run go import

Reply:
@andreaangiolillo  I have that running, I guess that's why it reformatted. What are you expecting this to be? Maybe something wrong with my formatter settings

Reply:
This is how it gets formatted :/ , tried go fmt as well after removing all new lines/gaps

Reply:
I don't know 😕  Thanks for the followup! I will resolve my comment